qa/rpc-tests/buip055.py

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import time
+import shutil
+import random
+from binascii import hexlify
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from test_framework.util import *
+from test_framework.script import *
+from test_framework.blocktools import *
+import test_framework.script as script
+import pdb
+import sys
+if sys.version_info[0] < 3:
+    raise "Use Python 3"
+import logging
+logging.basicConfig(format='%(asctime)s.%(levelname)s: %(message)s', level=logging.INFO)
+
+
+invalidOpReturn = hexlify(b'peer2peer cash requires larger blocks')
```

Comment:
need to be changed per spec, the new string is:
`Bitcoin: A Peer-to-Peer Electronic Cash System`
see: https://github.com/Bitcoin-UAHF/spec/blob/master/uahf-technical-spec.md#req-6-1-disallow-special-op_return-marked-transactions-with-sunset-clause